layer/contracts/runs.py

```diff
@@ -1,254 +1,15 @@
-import abc
-import hashlib
-import inspect
-import os
-import pickle  # nosec import_pickle
-import shutil
-import sys
-import uuid
-from dataclasses import dataclass, field, replace
-from pathlib import Path
-from typing import Any, Callable, Iterator, List, Optional, Sequence, Tuple, TypeVar
+from dataclasses import dataclass
+from typing import Callable, List
 
-import cloudpickle  # type: ignore
 from layerapi.api.entity.run_pb2 import Run as PBRun
 from layerapi.api.ids_pb2 import RunId
 
-from layer.config import DEFAULT_FUNC_PATH
-from layer.exceptions.exceptions import LayerClientException
-
-from .asset import AssetPath, AssetType
-from .fabrics import Fabric
 from .project_full_name import ProjectFullName
 
 
-def _language_version() -> Tuple[int, int, int]:
-    return sys.version_info.major, sys.version_info.minor, sys.version_info.micro
-
-
 GetRunsFunction = Callable[[], List[PBRun]]
```

Comment:
A bit unrelated but, was trying to understand this and it led me to `layer/tracker/output.py:def watch_get_runs(console: Console, get_runs_fn: GetRunsFunction) -> None:` which seems to be unused now. Something to clean up further.